etc/docker/docker_entrypoint.sh

```diff
@@ -84,12 +83,7 @@ NAT64_PREFIX=${NAT64_PREFIX/\//\\\/}
 sed -i "s/^prefix.*$/prefix $NAT64_PREFIX/" /etc/tayga.conf
 sed -i "s/dns64.*$/dns64 $NAT64_PREFIX {};/" /etc/bind/named.conf.options
 
-echo "Config:NCP:SocketPath \"$NCP_PATH\"" > /etc/wpantund.conf
-echo "Config:TUN:InterfaceName $TUN_INTERFACE_NAME " >> /etc/wpantund.conf
-echo "Daemon:SetDefaultRouteForAutoAddedPrefix $AUTO_PREFIX_ROUTE" >> /etc/wpantund.conf
-echo "IPv6:SetSLAACForAutoAddedPrefix $AUTO_PREFIX_SLAAC" >> /etc/wpantund.conf
-
-echo "OTBR_AGENT_OPTS=\"-I $TUN_INTERFACE_NAME\"" > /etc/default/otbr-agent
+echo "OTBR_AGENT_OPTS=\"-I $TUN_INTERFACE_NAME $NCP_PATH 115200\"" > /etc/default/otbr-agent
```

Comment:
```suggestion
echo "OTBR_AGENT_OPTS=\"-I '$TUN_INTERFACE_NAME' '$NCP_PATH' 115200\"" > /etc/default/otbr-agent
```

Reply:
That will break the service init as it will then treat `$NCP_PATH` for a separate command.